tests/ctr/ctr_test.go

```diff
@@ -38,3 +40,48 @@ func TestCtrSptUnikraft(t *testing.T) {
 func TestCtrSptRumprun(t *testing.T) {
 	t.Log(NotImplemented)
 }
+
+func TestCtrQemuUnikraftNginx(t *testing.T) {
+	pullParams := strings.Split("ctr image pull harbor.nbfc.io/nubificus/urunc/nginx-qemu-unikraft:latest", " ")
+	pullCmd := exec.Command(pullParams[0], pullParams[1:]...) //nolint:gosec
+	err := pullCmd.Run()
+	if err != nil {
+		t.Fatalf("Error pulling hello-hvt-rump:latest image: %v", err)
```

Comment:
The output error message should be fixed to reflect the correct image that we tried to pull.